### ISSUE TITLE
GBA

### ISSUE THREAD

**Issue by axelhei** [44670/BootNTR#87]
Gba VC titles wont show up on the viewer I don't know if it is possible to fix this.

**Comment by urherenow** [comment]
Nope. Never will be. A reboot into agb_firm or twl_firm is like hitting the reset button on your PC.
